GoogleTestAdapter/TestAdapter/TestDiscoverer.cs

```diff
@@ -47,22 +47,22 @@ public TestDiscoverer(ILogger logger, SettingsWrapper settings)
             CommonFunctions.LogVisualStudioVersion(_logger);
 
             _logger.LogInfo(Strings.Instance.TestDiscoveryStarting);
-            _logger.DebugInfo($"Solution settings: {_settings}");
+            _logger.DebugInfo(String.Format(Resources.Settings, _settings));
 
             try
             {
                 var reporter = new VsTestFrameworkReporter(discoverySink, _logger);
                 _discoverer.DiscoverTests(executables, reporter);
 
                 stopwatch.Stop();
-                _logger.LogInfo($"Test discovery completed, overall duration: {stopwatch.Elapsed}");
+                _logger.LogInfo(String.Format(Resources.TestDiscoveryCompleted, stopwatch.Elapsed));
             }
             catch (Exception e)
             {
-                _logger.LogError($"Exception while discovering tests: {e}");
+                _logger.LogError(String.Format(Resources.TestDiscoveryExceptionError, e));
             }
 
-            CommonFunctions.ReportErrors(_logger, "test discovery", _settings.DebugMode);
+            CommonFunctions.ReportErrors(_logger, Resources.TestDiscovery, _settings.DebugMode);
```

Comment:
Maybe you should pass down enum here rather than a string like you did with the other one.